BugReporter/Serialization/SerializableException.cs

```diff
@@ -177,7 +177,15 @@ public static SerializableException FromXmlString(string xml)
         {
             var serializer = new XmlSerializer(typeof(SerializableException));
             using StringReader reader = new(xml);
-            return (SerializableException)serializer.Deserialize(reader);
+            SerializableException exception = (SerializableException)serializer.Deserialize(reader);
+
+            if (exception.StackTrace is not null && exception.StackTrace.IndexOf("\r\n") < 0)
+            {
+                // Presume that the payload was serialized with \n only
+                exception.StackTrace = exception.StackTrace.Replace("\n", Environment.NewLine);
+            }
```

Comment:
Nit: I think `Environment.NewLine` should be used everywhere. Should we shorten the condition?
```suggestion
            if (exception.StackTrace?.IndexOf(Environment.NewLine) < 0)
            {
                // Presume that the payload was serialized with \n only
                exception.StackTrace = exception.StackTrace.Replace("\n", Environment.NewLine);
            }
```

Reply:
Yes, thank you.

Btw, https://github.com/dotnet/runtime/issues/40315 would have made this a non-issue